pep257.py

```diff
@@ -564,10 +566,15 @@ def main():
     Error.explain = options.explain
     Error.source = options.source
     collected = list(collected)
+    errors = check(collected, ignore=options.ignore.split(','))
     code = 0
-    for error in check(collected, ignore=options.ignore.split(',')):
+    count = 0
+    for error in errors:
         sys.stderr.write('%s\n' % error)
         code = 1
+        count += 1
+    if options.count:
+        print(count)
```

Comment:
Always happy to help. As maintainer of flake8-docstrings I think I'll have to account for this new flag in the next release because `--count` is a flag already offered by pep8. (It's what piqued my interest particularly. :))